scripts/data_contamination/README.md

```diff
@@ -65,3 +65,17 @@ with beam.Pipeline() as pipeline:
 ## Notes
 
 The beam script does not support outputting contaminated ngrams yet.
+
+
+## Docker
+
+To create and run docker image:
+
+    docker build  . -t contamination_script
+    docker run --input_path=<input_path> --scenario_data=<scenario_data> --output_stats=<output_stats> --input_format=<input_format> --rm -it  contamination_script:latest 
```

Comment:
This needs to be updated to use hyphens instead of underscores, such as `--input-path` instead of `--input_path`.

Reply:
Done